hash/wswan.xml

```diff
@@ -2212,15 +2209,15 @@ The GIZA chip used on a lot of cartridges is a sram voltage switch.
 		</part>
 	</software>
 
-	<!--
-	The Robot Works cartridge has an integrated infrared port for programming an
-	external robot (the Wonder Borg).
-	The Wonder Borg itself uses an Elan EM78P451AQ MCU (undumped).
-	-->
 	<software name="robworks" supported="partial">
 		<description>Wonder Borg Robot Works</description>
 		<year>2000</year>
 		<publisher>Bandai</publisher>
+		<notes><![CDATA[
+The Robot Works cartridge has an integrated infrared port for programming an
+external robot (the Wonder Borg).
+The Wonder Borg itself uses an Elan EM78P451AQ MCU (undumped).
+]]></notes>
```

Comment:
Not sure if I like that multi-line breaks indentation wrt the rest of the XML entity when opening in raw form. 
But if the general consensus is to go like this then I won't complain (would mean that front-ends should apply an `lstrip()` for each line, which is trivial to do but hey).

Reply:
Yeah, that is a bit of a downside with this. Also in json format it would look similarly ugly.